applications/ecallmgr/src/ecallmgr_call_events.erl

```diff
@@ -265,21 +260,20 @@ handle_cast({'transferer', _Props}, State) ->
 handle_cast({'b_leg_events', Events}, State) ->
     lager:debug("tracking b_leg events: ~p", [Events]),
     {'noreply', State#state{other_leg_events=Events}};
-handle_cast({'other_leg', _OtherLeg}, #state{other_leg_events=[]}=State) ->
-    lager:debug("ignoring other leg events for ~s", [_OtherLeg]),
-    {'noreply', State};
 handle_cast({'other_leg', OtherLeg}, #state{other_leg_events=_Events
                                             ,node=Node
                                            }=State) ->
     lager:debug("tracking other leg events for ~s: ~p", [OtherLeg, _Events]),
-    'true' = register_for_events(Node, OtherLeg),
+    _ = ecallmgr_call_sup:start_event_process(Node, OtherLeg),
```

Comment:
Why start this as a separate process?  I also don't follow why other_leg_events is being ignored here, lets go over this :)